generators/app/index.js

```diff
@@ -258,9 +274,21 @@ module.exports = class extends Generator {
     this.composeWith(require.resolve('../nsp'));
     this.composeWith(require.resolve('../eslint'));
 
+    if (!this.props.repositoryName) {
+      this.props.repositoryName = this.props.name;
+
+      if (this.props.repositoryName.startsWith('@')) {
+        this.props.repositoryName = this.props.repositoryName
```

Comment:
I think here we could also infer the default `githubAccount` value from the npm scope. (maybe set it as the default of the githubAccount prompt)